reviews/views.py

```diff
@@ -1,3 +1,72 @@
-from django.shortcuts import render
+import boto3
+import json
+import uuid
 
-# Create your views here.
+from django.http  import JsonResponse
+from django.views import View
+
+from core.utils     import login_required
+from my_settings    import AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY
+from arts.models    import Art
+from reviews.models import Review, ReviewImage
+
+class ReviewView(View):
+    @login_required
+    def post(self, request, art_id):
+        try:
+            user = request.user
+            art  = Art.objects.get(id=art_id)
+
+            review = Review.objects.create(
+                user_id = user.id,
+                art_id  = art.id,
+                comment = request.POST.get("comment")
+            )
+
+            s3_client = boto3.client(
+                's3',
+                aws_access_key_id = AWS_ACCESS_KEY_ID,
+                aws_secret_access_key = AWS_SECRET_ACCESS_KEY
+            )
+
+            image_file = request.FILES['image_file']
+            image_file_name = uuid.uuid1().hex
```

Comment:
uuid 잘 사용하셨습니다. str로 바꾸기 위함이였다면, str() method을 사용해도 좋을 것 같아요.
그리고 정렬은 꼭 확인해 주세요.
```suggestion
            image_file      = request.FILES['image_file']
            image_file_name = str(uuid.uuid1())
```

Reply:
네~ 정렬 수정했고 str() method로 수정했습니다, 감사합니다!